packages/backend/src/graphql/mutations/login-with-sgid/common.ts

```diff
@@ -0,0 +1,48 @@
+import { SgidClient } from '@opengovsg/sgid-client'
+
+import appConfig from '@/config/app'
+
+export const sgidClient = new SgidClient({
+  clientId: appConfig.sgid.clientId,
+  clientSecret: appConfig.sgid.clientSecret,
+  privateKey: appConfig.sgid.privateKey,
+  redirectUri: `${appConfig.webAppUrl}/login/sgid/redirect`,
+})
+
+export const SGID_COOKIE_NAME = 'plumber-sgid'
```

Comment:
can give a more specific variable and value? 
e.g.
SGID_MULTI_EMAIL_COOKIE = 'plumber-multi-email-sgid'

Reply:
or u can use ur multihat

Reply:
Hats are best, you got play TF2 before?